src/main/java/pinball/Renderer.java

```diff
@@ -69,21 +69,29 @@ private void renderCircle(Graphics2D graphics2D, Vector2 position, int radius) {
      * @param polygon the polygon to draw
      * @param position the center point of the polygon
      */
-    private void renderPolygon(Graphics2D graphics2D, PolygonShape polygon, Vector2 position) {
+    private void renderPolygon(Graphics2D graphics2D, PolygonShape polygon, Vector2 position, float angle) {
```

Comment:
So we know that this angle is in radians, change the name of the variable to `angleRadians`.